### DISCUSSION TITLE
Outreachy Intern

### DISCUSSION THREAD

**Discussion by anslemkelechi** [Open-Science-Community-Saudi-Arabia/Open-Science-Community-in-Saudi#28] [Outreachy 2022] [answered]
Hello Everyone, I am Anslem kelechi, an applicant in the outreachy internship program, I am passionate about open source and I am happy to be here. Looking forward to contributing the best way I can.

**Comment by BatoolMM** [comment, marked as answer]
Welcome to OSCSA, looking forward to your contributions!
